python-package/mlbox/encoding/categorical_encoder.py

```diff
@@ -9,7 +9,7 @@
 import os
 
 # Set the keras backend if not set, default is theano
-if not "KERAS_BACKEND" in os.environ:
+if "KERAS_BACKEND" not in os.environ:
     os.environ["KERAS_BACKEND"] = "theano"
 
 from keras.layers.core import Dense, Reshape, Dropout
```

Comment:
You can use `# noqa` at the end of the line to ignore pep8 errors for now.
We will discuss how to handle KERAS_BACKEND

```python
if "KERAS_BACKEND" not in os.environ:  # noqa
     os.environ["KERAS_BACKEND"] = "theano"  # noqa
```